manifests/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -136,7 +136,7 @@ spec:
           {{- else if .Values.global.configNamespace }}
             value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Values.global.configNamespace }}.svc:15012
           {{- else }}
-            value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.istio-system.svc:15012
+            value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Release.Namespace }}.svc:15012
```

Comment:
In this case - clearly not - again, the goal is to allow gateway to be deployed in separate NS ( so people with lower permissions can manage it ).